### PR TITLE
cargo fmt the code and tweak format rules

### PR DESCRIPTION
**Pull request by chao-mu**
Ran cargo fmt after removing rules per @kvark, two of which were because they needed nightly to run.

### CODE REVIEW

File: src/macros.rs [hunk 1]
```diff
@@ -33,36 +33,96 @@ macro_rules! vertex_attr_array {
 // For internal usage
 #[macro_export]
 macro_rules! vertex_format_size {
-    (Uchar2) => { 2 };
```
Comment (kvark) [inline src/macros.rs:36]: can we maybe change these to avoid `{` and `}` entirely, so that rustfmt doesn't put the value on a separate line?

Comment (chao-mu) [reply src/macros.rs:36]: How might you suggest things be changed to this effect? Also, I looked into rust-fmt settings and unfortunately while there exists something for this (format_macro_bodies=false) it is not stable yet.

Comment (kvark) [reply src/macros.rs:36]: I was thinking we could edit this to be `(Uchar2) => 2,` and so forth before/after formatting

Comment (chao-mu) [reply src/macros.rs:36]: That doesn't seem to be valid syntax, or I am doing something wrong. When I go to build, I get this error:
```
error: no rules expected the token `,`
  --> src/macros.rs:36:18
   |
36 |     (Uchar2) => 2,
   |                  ^ no rules expected this token in macro call

```
Also, doing 
```
(Uchar2) => (2);
```
Auto expands to the multiline {} when I save it with my editor which runs rust-fmt automatically (so presumably also with cargo fmt).

**Pull request merged**